docs/example-scenario/infrastructure/image-modeling.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Accelerate digital image-based modeling on Azure
+titleSuffix: Azure Example Scenarios
+description: Accelerate digital image-based modeling on Azure using Avere and Agisoft PhotoScan
+author: adamboeglin
+ms.date: 1/11/2019
+ms.custom: cat-team
+---
+
+# Accelerate digital image-based modeling on Azure
+
+This example scenario provides architecture and design guidance for any organization that wants to perform image-based modeling on Azure infrastructure as a service (IaaS). The scenario is designed for running photogrammetry software on Azure Virtual Machines (VMs) using high-performance storage that accelerates processing time. The environment can be scaled up and down as needed and supports terabytes of storage without sacrificing performance.
```

Comment:
While it's not in the style guide, convention is for "infrastructure as a service" to be "infrastructure-as-a-service".

Reply:
fixed.